nginx/nginx.conf

```diff
@@ -17,13 +16,18 @@ http {
   keepalive_timeout 15;
   types_hash_max_size 2048;
   client_max_body_size 20M;
-  include /etc/nginx/mime.types;
-  default_type application/octet-stream;
-  access_log /var/log/nginx/access.log;
-  error_log /var/log/nginx/error.log;
+  open_file_cache max=100;
   gzip on;
   gzip_disable "msie6";
+
+  ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
we should not advocate the use of TLSv1.0 https://www.lexiconn.com/blog/2015/12/pci-council-pushes-back-tls-1-0-end-of-life-date-to-june-2018/

Reply:
@0x13a good point, created issue #97